src/js/components/spinner.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import classnames from "classnames";
+
+const BASE_CLASS = "mw-spinner";
+
+export default class Spinner extends React.Component {
+
+	render() {
+		const classes = [
+			BASE_CLASS,
+			this.props.className
+		];
+
+		const style = { backgroundColor: this.props.color };
```

Comment:
fair! it's just my personal preference to change all styles with class names and assign all styling in the CSS, but I know React people are now all about mixing JS and CSS. whatever you want to do 😸